### ISSUE TITLE
Fix for broken notifications after High Sierra update

### ISSUE THREAD

**Issue by despecial** [citruspi/Spotify-Notifications#97]
Updated to latest version of Mac OS and it seems the notifications are broken.
Can you confirm/fix?

Thanks

**Comment by tobystokes** [comment]
I'm on 10.13.2 and works for me, freshly installed.

**Comment by despecial** [comment]
Had to re-install. Did not work after upgrading the OS. 
not sure what the problem is/was exactly. maybe file rights.

**Comment by despecial** [comment]
Still not working. It is sitting in the menu bar.
No notifications showing. Spotify is minified.

**Comment by dani-z** [comment]
Worked as expected for me, running on High Sierra 10.13.2

**Comment by c-lawlz** [comment]
I had the same problem, but after deleting the app (completely, using AppCleaner) and reinstalling it, it works fine.  Well... there's an annoying little Spotify logo next to each song name now, but I'm just happy it works :)

I'm wondering if not deleting the library files is why this did not work for @despecial

**Comment by despecial** [comment]
@c-lawlz yes, something was mixed up. I had to take the same steps to make the app working again. Plus, the old entry in the startup apps was broken and I had to remove it manually and after restarting Spotify Notifications, it added a new entry itself.

Was only a problem after High Sierra upgrade. But well, hopefully this thread might help others.
